yt/visualization/plot_modifications.py

```diff
@@ -1848,6 +1850,10 @@ class TimestampCallback(PlotCallback):
 
             "figure" -- the MPL figure coordinates: (0,0) is lower left, (1,1) is upper right
 
+    time_offset : float, (value, unit) tuple, or YTQuantity, optional
+        Apply an offset to the time shown in the annotation from the
+        value of the current time. Default: None, meaning no offset.
```

Comment:
can you say that this uses the `time_unit` kwarg for the unit if you pass in a scalar with no explicit units?